packages/hdwallet-tallyho/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@shapeshiftoss/hdwallet-tallyho",
+  "version": "1.19.0",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "dist/index.js",
+  "source": "src/index.ts",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsc --build",
+    "clean": "rm -rf dist tsconfig.tsbuildinfo",
+    "prepublishOnly": "yarn clean && yarn build"
+  },
+  "dependencies": {
+    "@shapeshiftoss/hdwallet-core": "1.19.0",
```

Comment:
This needs to be updated to the latest version. `1.21.2` is the latest.

Reply:
Merged main into this branch